scripts/windows-installer/wasmer.iss

```diff
@@ -17,8 +17,6 @@ DisableWelcomePage=no
 [Registry]
 Root: HKCU; Subkey: "Environment"; ValueType:string; ValueName: "WASMER_DIR"; \
     ValueData: "{app}"; Flags: preservestringtype
-Root: HKCU; Subkey: "Environment"; ValueType:string; ValueName: "WASMER_CACHE_DIR"; \
-    ValueData: "{app}\cache"; Flags: preservestringtype
```

Comment:
If not provided, `WASMER_CACHE_DIR` will be `WASMER_DIR + "/cache"`

In this case, I think we should go to the value that worked before:

```suggestion
Root: HKCU; Subkey: "Environment"; ValueType:string; ValueName: "WASMER_CACHE_DIR"; \
    ValueData: "{%USERPROFILE}\.wasmer\cache"; Flags: preservestringtype
```

Reply:
I think it's better not to set a register default value for it on Windows. We can keep the consistent usage style with Unix/Linux OS.

If users want to specify a cache directory, they can set an environment variable WASMER_CACHE_DIR, otherwise, env::temp_dir() will be used and it's safe for caching files.

/// Get the cache dir
pub fn get_cache_dir() -> PathBuf {
    match env::var("WASMER_CACHE_DIR") {
        Ok(dir) => {
            let mut path = PathBuf::from(dir);
            path.push(VERSION);
            path
        }
        Err(_) => {
            // We use a temporal directory for saving cache files
            let mut temp_dir = env::temp_dir();
            temp_dir.push("wasmer");
            temp_dir.push(VERSION);
            temp_dir
        }
    }
}
What do you think? @syrusakbary